README.md

```diff
@@ -1,3 +1,190 @@
+```
+
+5. Link the [ethereum-service Repository](https://github.com/codex-protocol/npm.ethereum-service)
+
+Make sure you've cloned the [ethereum-service repository](https://github.com/codex-protocol/npm.ethereum-service) and have `npm link`ed it so that the API will be able to use your locally-deployed smart contracts (see above).
+
+For more information on this repository, see the [README](https://github.com/codex-protocol/npm.ethereum-service/blob/master/README.md).
+
+6. Deploy [Codex Registry Smart Contracts](https://github.com/codex-protocol/contract.codex-registry)
+
+After you've set up Ganache and linked the [ethereum-service](https://github.com/codex-protocol/npm.ethereum-service) repository, you will need to deploy the Codex Registry smart contracts. Make sure Ganache is running, and then run:
+
+```bash
+$ cd contract.codex-registry
+$ npm run migrate:reset
+```
+
+This will make Truffle deploy the contracts to Ganache and copy over the built contract JSON files to the ethereum-service repository, where the API will be able to read them.
+
+
+Wow that was a lot. You should now be able to run the API. Give yourself a pat on the back if you made it this far!
+
+
+### Configure
+
+After you've installed & set up all dependencies, you should now update your `.env` file to match your local environment (for example, change the MongoDB or Redis ports if you're not using the defaults.) See the [Security](#security) section for more details on the `.env` file.
+
+
+## Usage
+
+1. Start [EEL](https://github.com/codex-protocol/service.eel):
+
+```bash
+$ cd service.eel
+$ npm start
+```
+
+2. Start the API:
+```bash
+$ cd service.codex-registry-api
+$ npm start
+```
+
+Now you should be able to make requests to the API! To check and see if it's responding to requests, try and hit [http://localhost:3001/etc/health-check](http://localhost:3001/etc/health-check). You should receive an `HTTP 200` response.
+
+
+## Maintainers
+
+- [John Forrest](mailto:john@codexprotocol.com) ([@johnhforrest](https://github.com/johnhforrest))
+- [Colin Wood](mailto:colin@codexprotocol.com) ([@Anaphase](https://github.com/Anaphase))
+- [Shawn Price](mailto:shawn@codexprotocol.com) ([@sprice](https://github.com/sprice))
+
+
+## Contribute
+
+If you have any questions, feel free to reach out to one of the repository [maintainers](#maintainers) or simply [open an issue](https://github.com/codex-protocol/service.codex-registry-api/issues/new) here on GitHub.
+
+[Pull Requests](https://github.com/codex-protocol/service.codex-registry-api/pulls) are not only allowed, but highly encouraged! All Pull Requests must pass CI checks (if applicable) and be approved by at least one repository [maintainer](#maintainers) before being considered for merge.
+
+Codex Labs, Inc follows the [Contributor Covenant Code of Conduct](https://contributor-covenant.org/version/1/4/code-of-conduct).
+
+
+## License
+
+[GNU Affero General Public License v3.0 or later](LICENSE) © Codex Labs, Inc
```

Comment:
I don't think this needs to include `or later`

Reply:
Well that's the name of the actual license we use, [as listed on SPDX](https://spdx.org/licenses/AGPL-3.0-or-later.html). It's different than ["GNU Affero General Public License v3.0 only"](https://spdx.org/licenses/AGPL-3.0-only.html).

Since we use the `AGPL-3.0-or-later` short identifier in our package.json, that's what I went with for the license section. We may want to consider using "3.0 only", not super sure of the differences.

Reply:
Hmm, then maybe we are using "3.0 only". Not super sure... I just went with what was instructed by the [standard-readme spec](https://github.com/RichardLitt/standard-readme/blob/master/spec.md#license) (says to use the SPDX identifier)